### PR TITLE
[PEPPER-1146] [Playwright] Minor update test participant-update-data

### PR DESCRIPTION
**Pull request by aweng98**
[PEPPER-1146](https://broadworkbench.atlassian.net/browse/PEPPER-1146)

[PEPPER-1146]: https://broadworkbench.atlassian.net/browse/PEPPER-1146?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: playwright-e2e/dsm/pages/participant-page/participant-page.ts [hunk 1]
```diff
@@ -106,7 +112,11 @@ export default class ParticipantPage {
     await input.fill(newValue);
 
     const updateButton = this.page.locator(this.getMainInputUpdateButtonXPath(inputEnum));
-    await updateButton.click();
+    await Promise.all([
+      waitForResponse(this.page, { uri: '/editParticipant'}),
+      waitForResponse(this.page, { uri: '/editParticipantMessageStatus'}),
+      updateButton.click()
+    ]);
```
Comment (aweng98) [inline playwright-e2e/dsm/pages/participant-page/participant-page.ts:119]: waits for update requests to finish when click `Update` button.

File: playwright-e2e/tests/dsm/participant/participant-update-data.spec.ts [hunk 2]
```diff
@@ -2,19 +2,15 @@ import { expect } from '@playwright/test';
...
 import { MainInfoEnum } from 'dsm/pages/participant-page/enums/main-info-enum';
 import { AdditionalFilter, CustomViewColumns } from 'dsm/component/filters/sections/search/search-enums';
 import { waitForNoSpinner } from 'utils/test-utils';
 
 test.describe('Editig Participant Information', () => {
-  let participantPage: ParticipantPage;
-
   let shortID: string;
   let firstName: string;
-  let lastName: string;
```
Comment (aweng98) [inline playwright-e2e/tests/dsm/participant/participant-update-data.spec.ts:17]: not used.

File: playwright-e2e/dsm/component/tables/participant-list-table.ts [hunk 3]
```diff
@@ -40,8 +39,7 @@ export class ParticipantListTable extends Table {
 
   public async openParticipantPageAt(position: number): Promise<ParticipantPage> {
     await this.getParticipantAt(position).click();
-    await waitForNoSpinner(this.page);
-    await this._participantPage.assertPageTitle();
```
Comment (aweng98) [inline playwright-e2e/dsm/component/tables/participant-list-table.ts:44]: Moved this to the generic func `waitForReady`.

File: playwright-e2e/tests/dsm/participant/participant-update-data.spec.ts [hunk 4]
```diff
@@ -2,19 +2,15 @@ import { expect } from '@playwright/test';
...
 import { StudyEnum } from 'dsm/component/navigation/enums/selectStudyNav-enum';
 import ParticipantListPage from 'dsm/pages/participant-list-page';
-import ParticipantPage from 'dsm/pages/participant-page/participant-page';
 import { logInfo } from 'utils/log-utils';
 import { faker } from '@faker-js/faker';
 import { MainInfoEnum } from 'dsm/pages/participant-page/enums/main-info-enum';
 import { AdditionalFilter, CustomViewColumns } from 'dsm/component/filters/sections/search/search-enums';
 import { waitForNoSpinner } from 'utils/test-utils';
 
 test.describe('Editig Participant Information', () => {
```
Comment (Rinchan20) [inline playwright-e2e/tests/dsm/participant/participant-update-data.spec.ts]: could we use this chance to fix Editig to Editing?

Comment (aweng98) [reply playwright-e2e/tests/dsm/participant/participant-update-data.spec.ts]: Good catch. Fixed typo.

**Review by Rinchan20 — APPROVED** [review]
passes for me in dev and test

**Pull request merged**